tools/representations.py

```diff
@@ -460,6 +500,8 @@ def __str__(self):
             AUTHENTICATION_INFO: self.authentication_info_url,
             API_KEY_PARAMETER_NAME: self.api_key_parameter_name,
             LICENSE: self.license_url,
+            FEATURES: self.features,
+            STATUS: self.status,
         }
         return json.dumps(self.schematize(**attributes), ensure_ascii=False)
 
```

Comment:
```python
def has_status(self, status):
    return self.status == status or (self.status is None and status == ACTIVE)
```

> Where ACTIVE should be added to constants.py as ACTIVE = "active" and imported in this file.

Done

Reply:
```python
def has_feature(self, feature):
    static_sources = self.get_static_sources(self.static_reference)
    return any(
        feature in static_source.features
        for static_source in static_sources
    )
```
...I think we want to search if feature is either in realtime or its associated static sources, like:

```python

    def has_feature(self, feature):
        static_sources = self.get_static_sources(self.static_reference)
        in_static_source = any(
            feature in static_source.features
            for static_source in static_sources
        )
        in_realtime_source = feature in self.features
        return in_static_source or in_realtime_source

```

Reply:
I think it could be something like that indeed. I've asked for feedback from the community [in the original thread](https://mobilitydata-io.slack.com/archives/CSP7HDF37/p1656957145260369?thread_ts=1655335737.995849&cid=CSP7HDF37). Based on the feedback, I think we will be able to implement the most useful solution.

Reply:
We will go with option 3 listed [here](https://mobilitydata-io.slack.com/archives/CSP7HDF37/p1656957145260369?thread_ts=1655335737.995849&cid=CSP7HDF37), so the changes you implemented are all good.